layouts/book-a-demo/single.html

```
@@ -0,0 +1,273 @@
+    <script type="text/javascript" src="{{ $bookDemoJs.RelPermalink }}" defer></script>
+    {{ $altchaJs := resources.Get "js/altcha/altcha.js" }}
+    <script type="module" type="text/javascript" src="{{ $altchaJs.RelPermalink }}" defer></script>
+    {{ $altchaWorkerJs := resources.Get "js/altcha/worker.js" }}
+    <script type="module" type="text/javascript" src="{{ $altchaWorkerJs.RelPermalink }}" defer></script>
+  {{ else }}
+    {{ $newsletterJs := resources.Get "js/newsletter.js" | minify | fingerprint }}
+    <script type="text/javascript" src="{{ $newsletterJs.RelPermalink }}" integrity="{{ $newsletterJs.Data.Integrity }}" defer></script>
+    {{ $bookDemoJs := resources.Get "js/bookdemo.js" | minify | fingerprint }}
+    <script type="text/javascript" src="{{ $bookDemoJs.RelPermalink }}" integrity="{{ $bookDemoJs.Data.Integrity }}" defer></script>
+    {{ $altchaJs := resources.Get "js/altcha/altcha.js" }}
+    <script type="module" type="text/javascript" src="{{ $altchaJs.RelPermalink }}" integrity="{{ $altchaJs.Data.Integrity }}" defer></script>
```

HTMLHint (1.5.0)

[error] Duplicate of attribute name [ type ] was found.

(attr-no-duplication)